cmds/core/which/which_test.go

```diff
@@ -43,26 +43,21 @@ var (
 	p = os.Getenv("PATH")
 )
 
-func setup() error {
+func setup(t *testing.T) {
 	var err error
 
```

Comment:
Should we call this 
TestSetup
and call
t.Helper() here? 

I really don't know best practice ... whatever seems best.

Reply:
I think we can name helpers as we want, see this for reference https://go.dev/wiki/TestComments#mark-test-helpers, also if we name it TestSetup it will become a test.